requirements.txt

```diff
@@ -2,3 +2,6 @@
 bokeh==1.1.0
 # web_widget_plotly_chart
 plotly==4.1.0
+# web_widget_mpld3_chart
+matplotlib>=2.0.0
+mpld3>=0.3
```

Comment:
Aren't you adding in the module a specif version of the library? You should freeze it here.